projects/stonent.md

```diff
@@ -0,0 +1,47 @@
+## Project Name
+Stonent
+## Category 
+Application
+
+## Project Description
+Stonent helps NFT-artists fight with intellectual property theft. We use neural network to detect plagiarism of NFT ERC1155 tokens directly in blockchain using Chainlink oracles.
+
+## Use of IPFS, Filecoin and Libp2p
+We download NFTs from IPFS and indexed it via NN.
+
+## Project Status
+under development
+
+## Previews
+DAPP with originality score ![image](https://user-images.githubusercontent.com/31879791/117173733-bdddc880-add5-11eb-9eb3-b74e9ba6db98.png)
+
+
+## Target Audience
+NFT artist and NFT buyers 
+
+## Rough estimated user base (if applicable)
+0-10
+
+## Github repo
+https://github.com/vladimir3322/stonent_project
+
+## Website
+http://stonent.us-east-2.elasticbeanstalk.com/
+
+## Docs
+https://github.com/vladimir3322/stonent_project/blob/main/Readme.md
+
+### Team Size  
+2-3
+
+### Team members  
+Stepan Samsonov
+
+Vladimir Tidva
+
+## How the community can engage
```

Comment:
Need to create a discussion thread in this repo's Discussion section and link it here!